server/src/controllers/UserChapterRole/resolver.ts

```diff
@@ -0,0 +1,37 @@
+import { Arg, Ctx, Int, Mutation, Resolver } from 'type-graphql';
+import { GQLCtx } from 'src/common-types/gql';
+import { Event, UserChapterRole } from 'src/models';
+
+@Resolver()
+export class UserChapterRoleResolver {
+  @Mutation(() => UserChapterRole)
```

Comment:
Just checking, we're returning this so we can display a _interested_ toggle, is that right?

Reply:
To be honest, I returned to show that data is updated, but may be we can use it to display the interested toggle .